pytket/extensions/qiskit/backends/ibm.py

```diff
@@ -519,6 +540,11 @@ def process_circuits(
         postprocess = kwargs.get("postprocess", False)
         simplify_initial = kwargs.get("simplify_initial", False)
 
+        options: Options = kwargs.get("options")
+        if options is None:
+            options = self._sampler_options
+        options.execution.shots = n_shots
```

Comment:
The test is failing because `n_shots` here is different from `n_shots` below (where code was deleted). I think `options.execution_shots` needs to be set further down (when submitting an individual circuit).

Reply:
I have moved this lower down. The n_shots property is now set per chunk of a batch.